conf/iCubFindDependencies.cmake

```diff
@@ -31,9 +31,13 @@ message(STATUS "CMake modules directory: ${CMAKE_MODULE_PATH}")
 # This is a workaround to enable the warning only if
 # icub_firmware_shared was found, but the version is not compatible with
 # the requested one.
-find_package(icub_firmware_shared COMPONENTS canProtocolLib QUIET)
+find_package(icub_firmware_shared QUIET)
 if(icub_firmware_shared_FOUND)
-  find_package(icub_firmware_shared 4.0.7 COMPONENTS canProtocolLib)
+    find_package(icub_firmware_shared EXACT 1.15.100)
```

Comment:
This will fail with icub_firmware_shared 1.15.101, and given that we have a more complex version check logic under i guess this is not what you intended.

Reply:
No as we discussed off-line the second find_package was wrong, I removed thanks!